string class_name
float32 probability
geometry_msgs/Pose pose
float32 width
